stdlib/public/runtime/HeapObject.cpp

```diff
@@ -235,11 +235,12 @@ BoxPair::Return swift::swift_makeBoxUnique(OpaqueValue *buffer, const Metadata *
   }
 }
 
-BoxPair::Return swift::swift_allocBox(const Metadata *type) {
+BoxPair swift::swift_allocBox(const Metadata *type) {
   return _swift_allocBox(type);
 }
 
-static BoxPair::Return _swift_allocBox_(const Metadata *type) {
+SWIFT_CC(swift)
+static BoxPair _swift_allocBox_(const Metadata *type) {
```

Comment:
I'm fairly certain it's correct to use `swiftcall` here, but it's worth noting that this method was added since the original PR.

Reply:
All three of the allocBox pieces should be `swiftcall`: the `_swift_allocBox` function pointer and the functions `swift_allocBox` and `_swift_allocBox_`. 

Having said that, I see that Instruments does not actually use the `_swift_allocBox` function pointer which is the only reason for the extra function and function pointer to exist. We can mark `swift_allocBox` as `swiftcall` and delete the other two.

Reply:
I'm planning to remove all of these function pointers later. For now we can just fix the existing `swift_allocBox` pieces in place.

Reply:
I've removed the function pointer and simplified it to just the implementation. Unless I'm mistaken (and it's entirely possible I am), we shouldn't need to repeat the `SWIFT_CC(swift)` before the implementations since it's already in the header files.

Reply:
Or delete them now. That's good too.

Reply:
There is also a function pointer declaration in InstrumentsSupport.h.

Reply:
I removed the `BoxPair (*_swift_allocBox)(Metadata const *type)` declaration there in the same commit.

Reply:
Apparently I can't read header files today. I missed that and your earlier change to swift_allocBox in HeapObject.h. Everything looks good now.